modules/activiti-engine/src/test/java/org/activiti/engine/test/bpmn/subprocess/CallActivityTest.java

```diff
@@ -228,5 +231,47 @@ protected BpmnModel loadBPMNModel(String bpmnModelFilePath) throws Exception {
     return bpmnModel;
   }
 
+  public void testCallActivityWithBusinessKey() throws Exception {
+    BpmnModel mainBpmnModel = loadBPMNModel(BUSINESS_KEY_TEST_MAIN_PROCESS_RESOURCE);
+    BpmnModel childBpmnModel = loadBPMNModel(BUSINESS_KEY_TEST_CHILD_PROCESS_RESOURCE);
+    BpmnModel notInheritBusinessKeyMainBpmnModel = loadBPMNModel(NOT_INHERIT_BUSINESS_KEY_MAIN_PROCESS_RESOURCE);
+    BpmnModel inheritBusinessKeyMainBpmnModel = loadBPMNModel(INHERIT_BUSINESS_KEY_MAIN_PROCESS_RESOURCE);
+
+    Deployment childDeployment = processEngine.getRepositoryService()
+      .createDeployment()
+      .name("childProcessDeployment")
+    .addBpmnModel("childProcess.bpmn20.xml", childBpmnModel).deploy();
 
+    Deployment masterDeployment = processEngine.getRepositoryService()
+      .createDeployment()
+      .name("masterProcessDeployment")
+    .addBpmnModel("masterProcess.bpmn20.xml", mainBpmnModel).deploy();
+
+    Deployment notInheritBusinessKeyMainDeployment = processEngine.getRepositoryService()
+            .createDeployment()
+            .name("notInheritBusinessKeyMainDeployment")
+          .addBpmnModel("masterProcess.bpmn20.xml", notInheritBusinessKeyMainBpmnModel).deploy();
+
+    Deployment nheritBusinessKeyMainDeployment = processEngine.getRepositoryService()
+            .createDeployment()
+            .name("nheritBusinessKeyMainDeployment")
+          .addBpmnModel("masterProcess.bpmn20.xml", inheritBusinessKeyMainBpmnModel).deploy();
+
+    // No use of business key attributes
+    ProcessInstance pi = runtimeService.startProcessInstanceByKey("mainProcess");
+    ProcessInstance subProcessInstance = runtimeService.createProcessInstanceQuery().superProcessInstanceId(pi.getId()).singleResult();
+    assertNull(subProcessInstance.getBusinessKey());
+    
+    // Modeled using expression: businessKey="${busKey}"
+    Map<String,Object> variables = new HashMap<>();
```

Comment:
this isn't supported in java 1.6

Reply:
@balsarori 
Oh... I'm sorry.
Should I fix this code by another pull request?

Reply:
Thanks. It seems this bug has been already fixed with the following commitment.
https://github.com/Activiti/Activiti/commit/d4723ab5626b1fb4a069e793055116875f3da547